database_config.sql

```diff
@@ -27,7 +28,19 @@ CREATE TABLE `UserData` (
     `PricePaidPerShare` FLOAT
 );
 
+CREATE TABLE `Logs` (
+    `ID` INT,
+    FOREIGN KEY (`ID`) REFERENCES `Users` (`ID`) ON DELETE CASCADE,
```

Comment:
Remove 'ON DELETE CASCADE' since we don't want the entries to be deleted when a user deletes their account. This may cause some issues with MySQl. If so, let me know and I'll figure it out.